acceptance/lib/helper.rb

```diff
@@ -30,7 +30,8 @@ def self.initialize_config(options)
     puppet_build_version = get_option_value(options[:puppet_build_version],
                          nil, "Puppet Agent Development Build Version",
                          "PUPPET_BUILD_VERSION",
-                         "ba39f2f2fd1b4def9a7f366d469acb1c5f6db824", :string)
+                         "5cb5a63982c01f3ed0182c25b61213d208573d8d",
```

Comment:
I was following the confluence page.  It doesn't mention that.  :(  But it seems obviously wrong to have that variable defined and pointing to an incorrect value.

@kurtwall whatever happened when you were looking into getting rid of that?